testcontainers-common/src/main/java/com/playtika/test/common/utils/ContainerUtils.java

```diff
@@ -123,19 +169,24 @@ public static int getAvailableMappingPort() {
         }
     }
 
-    public static Consumer<OutputFrame> containerLogsConsumer(Logger log) {
+    /**
+     * Default logs consumer (trim messages, log with debug level, don't log last message if empty)
+     * @return Consumer for {@link GenericContainer#withLogConsumer(java.util.function.Consumer)}
+     */
+    public static Consumer<OutputFrame> containerLogsConsumer(Logger logger) {
         return (OutputFrame outputFrame) -> {
+            String trimmed = outputFrame.getUtf8String().trim();
             switch (outputFrame.getType()) {
                 case STDERR:
-                    log.debug(outputFrame.getUtf8String());
+                    logger.error(trimmed);
```

Comment:
Have you checked that this works as expected? It pollutes consumer logs with container logs even for normal startup.

Reply:
I will change it back.
In your experience many containers/servers log debug messages to stderr?
For your reference, here is the [upstream implementation](https://github.com/testcontainers/testcontainers-java/blob/1699e4708b15fcbf26b80f65fbbee48fa0b0c4d0/core/src/main/java/org/testcontainers/containers/output/Slf4jLogConsumer.java#L69).

Reply:
I've checked several, and for all of them there was such issue, meaning that at least for some of the containers we will have log pollution.